code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -100,7 +100,7 @@
 	                                                  // (also affects temperature processing)
 
 	var/heat_discomfort_level = 315                   // Aesthetic messages about feeling warm.
-	var/cold_discomfort_level = 285                   // Aesthetic messages about feeling chilly.
+	var/cold_discomfort_level = 295                   // Aesthetic messages about feeling chilly.
```

Comment:
That is **BODY** temperature. It works on a differential, so to get bodytemp to 21 C you need something at least in the 10C range. Optimistically; the mild snow stabilizes bodytemp at about 283K only standing still (moving actually raises bodytemp, TMYK) and it's almost -10C. Mind you, stadard atmos probably warmed them a bit, but it will too on unsim/planetsim.

It had been absurd that you could have mobs' bodies be chilled well to sub-zero temperatures before they so much as get uncomfortably cold.